src/stk/databases/mongo_db/value.py

```diff
@@ -168,7 +168,20 @@ def put(self, molecule, value):
         return self._put(HashableDict(json))
 
     def _put(self, json):
-        self._values.insert_one(json)
+        keys = dict(json)
+        keys.pop('v')
```

Comment:
What is 'v'?

Reply:
This comes from JSON entry actually deposited into the MongoDB. The entry deposited into the MongoDB has the form 

```python

{
    "v": 12
     "KeyName1": "KeyValue1"
    "KeyName2": "KeyValue2"
   # On so on.
}
```

So v is just the value stored in the ValueDB for that molecule. I use `v` instead of `value` to keep the entry as small as possible and therefore keep the MonoDB size as small as possible too.

Reply:
This should have a comment to explain what 'v' is.